app/components/forms/orders/order-form.js

```diff
@@ -53,10 +53,12 @@ export default Component.extend(FormMixin, {
     run.later(() => {
       const currentTime = moment();
       const diff = moment.duration(willExpireAt.diff(currentTime));
-      this.set('getRemainingTime', moment.utc(diff.asMilliseconds()).format('mm:ss'));
       if (diff > 0) {
+        this.set('getRemainingTime', moment.utc(diff.asMilliseconds()).format('mm:ss'));
         this.timer(willExpireAt, orderIdentifier);
       } else {
+        this.set('getRemainingTime', '00:00');
```

Comment:
Why?

Reply:
bcz if we go to order/new page again after order is expired, timer again set 'getRemainingTime which is different value and show this value for a sec and should show 00.00, and then transition to order/expired page so instead of showing diff value , should show 00.00

Reply:
Correct